### DISCUSSION TITLE
H96 MAX X3 stucks on the stock box Logo only when i do reboot from the cli

### DISCUSSION THREAD

**Discussion by 833M0L3** [devmfc/debian-on-amlogic#86] [Q&A] [answered]
Hi, I have this strange issue with my H96 Max X3 S905X3. The box boots completely fine when I plug in the power. But when I do reboot command from the cli, it turns off, H96 Stock Boot shows and stuck there. But if I turn off the power and turn it on, it boots normally. Any idea what might be the issue?

![image](https://github.com/user-attachments/assets/12c6c825-a62e-4c08-a683-bb74fc14df67)

**Comment by devmfc** [comment, marked as answer]
If you really have stock u-boot, I would suspect one of the following:
- poor quality power supply 
- bad SD card if you're booting from that. Try to boot from USB flash drive of another SD card

Otherwise I don't know and you'll need a serial log from booting to investigate further.

**Reply by 833M0L3** [reply]
Yes, it was a poor quality power supply. After I connected it to another one the issue stopped happening. Thanks for the tips.

Btw, I also have HK1 Box with s905x3, when I install your image, the system shows it as H96 MAX. Any idea why? I am using the correct config name in the boot config. 
![image](https://github.com/user-attachments/assets/09632d62-8927-4ec0-9f61-6ad70914cebb)
![image](https://github.com/user-attachments/assets/c6c60b6b-4ab3-4965-8bce-4c43b5a5f6b9)

**Reply by devmfc** [reply]
Because they are (or at least were at some point in time) exactly the same.
